mocks/docker-compose.yml

```diff
@@ -55,6 +55,20 @@ services:
     networks:
       - vro_intranet
 
+  mock-bie-kafka:
+    profiles: ["all","kafka"]
```

Comment:
Would it be worthwhile to also have a "bie" profile in addition to "kafka"? We could use kafka for any future kafka service, and "bie" would be specific to that use case.

Reply:
I considered it but wanted to keep it short. Didn't want to use just `bie` since BIE may have other services. I could be convinced otherwise, plus we can change it later.

Reply:
No need to change it now! We can always deal with that later! Not a hard change!